AmplifyPlugins/Storage/Tests/AWSS3StoragePluginTests/AWSS3StoragePluginAsyncBehaviorTests.swift

```diff
@@ -11,24 +11,45 @@ import Amplify
 @testable import AmplifyTestCommon
 @testable import AWSPluginsTestCommon
 
-class AWSS3StoragePluginAsyncBehaviorTests: AWSS3StoragePluginTests {
+class AWSS3StoragePluginAsyncBehaviorTests: XCTestCase {
```

Comment:
In the spirit of [locality](https://en.wikipedia.org/wiki/Locality_of_reference), I'm decoupling this from the parent class which added little value (this class was overriding most of its work).